app/code/Magento/Multishipping/Test/Mftf/Test/StorefrontMultishippingUpdateProductQtyTest.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontMultishippingUpdateProductQtyTest">
+        <annotations>
+            <features value="Multishipping"/>
+            <stories value="Checkout with multiple addresses."/>
+            <title value="Update Product Quantity on Ship to Multiple Addresses Page."/>
+            <description value="Verify customer will see correct product quantity after return to Ship to Multiple Addresses from Shipping information page."/>
+            <severity value="MAJOR"/>
+            <testCaseId value="MC-41697"/>
+            <useCaseId value="MC-40021"/>
+            <group value="multishipping"/>
+        </annotations>
+
+        <before>
+            <!--Create test data.-->
```

Comment:
@eduard13 it's good practice to use comments to make test more clear to understand.
They appeared as actions *without stepkey* and I suppose are not confusing but make tests more readable.

Reply:
Removed

Reply:
@zakdma I agree that in some cases, when we do some complex actions it may be useful to have some comments, but if we have some simple steps (login/logout admin/customer) or just creating/removing something, then comments are only duplicating the info that is provided by `stepKey`.